mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_759 net/minecraft/client/render/item/HeldItemRenderer
+	FIELD field_21807 MAP_BACKGROUND Lnet/minecraft/class_1921;
+	FIELD field_21808 MAP_BACKGROUND_CHECKERBOARD Lnet/minecraft/class_1921;
```

Comment:
Use regular word order (`CHECKERBOARD_MAP_BACKGROUND`).

Reply:
These are the names :mojang: used for the texture these layers are using. Should I really change them?

Reply:
Also swapping them would make it... weird.

This is the map background's checkerboard pattern.

Reply:
> This is the map-background's checkerboard pattern.